### PR TITLE
feat(deps): Add Nextcloud 28 support

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

**Comment by ChristophWurst** [comment]
needs info.xml update

**Comment by nickvergessen** [comment]
> needs info.xml update

Yeah, I skipped it because you still want to send the final before that I guess.

**Comment by ChristophWurst** [comment]
After https://github.com/nextcloud/mail/pull/8482 the app will be branched off for 27

File: appinfo/info.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
 	<screenshot>https://user-images.githubusercontent.com/1374172/79554966-278e1600-809f-11ea-82ea-7a0d72a2704f.png</screenshot>
 	<dependencies>
 		<php min-version="8.0" max-version="8.2" />
-		<nextcloud min-version="26" max-version="27" />
+		<nextcloud min-version="27" max-version="28" />
```
Comment (GretaD) [inline appinfo/info.xml:33]: now that im thinking, should i change the min-version to 27? or should i leave it 26?

Comment (ChristophWurst) [reply appinfo/info.xml:33]: Leave 26 until it EOLs or we need a 27 API

**Pull request merged**